Add the vue standard linter to the eslint config

blascsi opened the pull request:
I have added the @vue/eslint-config-standard to the extends in the .eslintrc.js.
`npm run lint` found no lint errors after this, so I made no changes to the components.

Hope this helps!

Resolves #1

Fohlen commented:
Hey @blascsi 
You PR looks good!

I haven't set up Travis so far, so I need to test the linting when I use `dashboard` this weekend / next week.
So, it'll take a few days, no worries, your PR won't stal.

All the best, greetings from Tuebingen to lovely Budapest 🍺

blascsi commented:
Great, thank you @Fohlen !
Hope everything will be fine :)

blascsi commented:
Hi @Fohlen 
Since then I have found this [article](https://eslint.vuejs.org/user-guide/#usage) which indicated that the way I added the linter was incorrect. I have corrected the error, now the linter has found errors, and autofixed most of them :)

blascsi commented on src/components/modals/location/Delete.vue:
```diff
@@ -27,7 +54,8 @@ export default {
     name: {
-      type: String
+      type: String,
+      default: '',
```
Please check this line, the linter required a default value here, but I wasn't sure what to set (especially in german :smile: )

Fohlen replied:
I believe this can be solved by removing the `default` and replacing it with `required: true`

blascsi commented on src/components/modals/source/Delete.vue:
```diff
@@ -27,7 +54,8 @@ export default {
     url: {
-      type: String
+      type: String,
+      default: '',
```
The same here as above.

Fohlen commented:
Hey @blascsi how are you invoking the linter? Using `npm run lint` will not issue any `linter` errors on the mentioned passage.

blascsi commented:
I changed the default values to required :smile: Sorry this is my first Vue PR!
And there shouldn't be any lint errors, the linter was autofixing most of them, only the two errors where I made the comments were fixed by hand.

Fohlen commented:
This PR looks good now! Will merge. Thanks for contributing ✨ 

You can add yourself to the [list of collaborators](https://github.com/eventzimmer/schema/blob/master/CONTRIBUTORS.md), as you like.

Greetings from Vienna!

blascsi commented:
Awesome! Thank you and greetings from Budapest! Have a great day!
